src/data/nat/prime.lean

```diff
@@ -593,6 +594,27 @@ lemma mem_factors {n p} (hn : 0 < n) : p ∈ factors n ↔ prime p ∧ p ∣ n :
 ⟨λ h, ⟨prime_of_mem_factors h, (mem_factors_iff_dvd hn $ prime_of_mem_factors h).mp h⟩,
  λ ⟨hprime, hdvd⟩, (mem_factors_iff_dvd hn hprime).mpr hdvd⟩
 
+lemma factors_subset {n k : ℕ} (h : k ≠ 0) : n.factors ⊆ (n * k).factors :=
```

Comment:
Should this be `factors_subset_left`?

Reply:
what would the left refer to? I'm right-multiplying so should be right I guess? it seemed to me like the most "natural" phrasing of it, though, which is why I didn't specify which side

Reply:
I feel like multiplying on the left is also quite common, e.g. through `smul`, so let's at least specify the side:
```suggestion
lemma factors_subset_right {n k : ℕ} (h : k ≠ 0) : n.factors ⊆ (n * k).factors :=
```